src/darwin/Framework/CHIP/MTRDeviceController_Internal.h

```diff
@@ -69,13 +69,11 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, readonly) NSMapTable<NSNumber *, MTRDevice *> * nodeIDToDeviceMap;
 @property (readonly, assign) os_unfair_lock_t deviceMapLock;
 
-@property (readwrite, nonatomic) NSUUID * uniqueIdentifier;
```

Comment:
Still need this to use self.

Reply:
But why do we want to use self for the set?  This thing should be set once at init time, never modified again....

Reply:
I'd like us to move to objective C patterns, where we use properties for this, so we can fully utilize `@synchronized` and atomic properly.

Reply:
OK.  Well, we can put the readwrite property into the `@implementation`, at least.  I'll do that.

Reply:
Moved into `@implementation`